File: IS.Api/IS.Api/Models/RequestModels/DepartmentRequestModel.cs
using IS.Api.DataAccess;
using Microsoft.AspNetCore.Mvc.Rendering;
using System.ComponentModel.DataAnnotations;

namespace IS.Api.Models
{
    public class DepartmentRequestModel
    {
        public string FunctionID { get; set; }
        public Department_MST department { get; set; }
        public ClientInformation client { get; set; }
    }
}

File: IS.Web.Angular/src/app/models/requestmodels/departmentrequestmodel.ts
import { GlobalConstants } from "src/app/common/globalconstants.model";
import { ClientInformation } from "../clientinformation";
import { Department } from "../department.model";

export class DepartmentRequestModel {
    public functionID:string=GlobalConstants.EMPTY_STRING;
    public department:Department=new Department();
    public client:ClientInformation=new ClientInformation();
}
